vtex-apps/store-components
# Configurable thumbnail dimensions in product gallery

#### What problem is this solving?

This PR adds the ability to configure the width and height of product thumbnails in the carousel component. By allowing control over the thumbnail dimensions, this can improve page performance and SEO by ensuring that appropriately sized images are used, reducing unnecessary data loading.

#### How to test it?

You can test this feature by setting custom values for thumbCustomWidth and thumbCustomHeight in the product image gallery component. Ensure that thumbnails are resized correctly based on these values, and monitor the impact on page load times and SEO scores.

For test use this [Workspace](https://pr31410--italiaonlineqa.myvtex.com)

#### Screenshots or example usage:

In production:
Official component in production unable to set thumbnail size.
See props: [Product Image](https://developers.vtex.com/docs/apps/vtex.store-components/ProductImages)

In workspace:

<img width="1492" alt="Screenshot 2024-09-17 alle 16 25 54" src="https://github.com/user-attachments/assets/8bdc99ca-1250-4519-836f-98c3ba9fe58d">

can be able to set values in px from props component:
<img width="518" alt="Screenshot 2024-09-17 alle 16 43 40" src="https://github.com/user-attachments/assets/671f16b3-9901-4813-b473-8eabdd6ee300">


#### Describe alternatives you've considered, if any.

An alternative approach considered was to use CSS only for thumbnail resizing, but this would not allow for dynamic configuration through props or tailored image dimensions for performance.

#### Related to / Depends on

No dependencies.
